packages/gatsby/src/utils/get-ssl-cert.ts

```diff
@@ -1,17 +1,30 @@
-const report = require(`gatsby-cli/lib/reporter`)
-const fs = require(`fs`)
-const path = require(`path`)
-const os = require(`os`)
+import report from "gatsby-cli/lib/reporter"
+import fs from "fs"
+import path from "path"
+import os from "os"
+import { ICert } from "../commands/types"
 
-const absoluteOrDirectory = (directory, filePath) => {
+const absoluteOrDirectory = (directory: string, filePath: string): string => {
   // Support absolute paths
   if (path.isAbsolute(filePath)) {
     return filePath
   }
   return path.join(directory, filePath)
 }
 
-module.exports = async ({ name, certFile, keyFile, directory }) => {
+interface IGetSslCertArgs {
+  name: string
+  certFile?: string
+  keyFile?: string
+  directory: string
+}
+
+export const getSslCert = async ({
```

Comment:
If we're changing this, can you change it to a function?

```suggestion
export async function getSslCert({
```

Reply:
fixed